scripts/configure-bedrock-access.sh

```
@@ -0,0 +1,186 @@
+
+  local -a aws_cli_opts
+  aws_cli_opts=("--region" "$aws_region")
+  if [[ -n "$aws_profile" ]]; then
+    aws_cli_opts+=("--profile" "$aws_profile")
+  fi
+
+  local trust_policy_file
+  local inline_policy_file
+  trust_policy_file="$(mktemp)"
+  inline_policy_file="$(mktemp)"
+  trap "rm -f '$trust_policy_file' '$inline_policy_file'" EXIT
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)